packages/trader/hardhat.config.ts

```diff
@@ -1,6 +1,9 @@
-const { getHardhatConfig } = require("@uma/common");
+import "@nomiclabs/hardhat-web3";
+import "@nomiclabs/hardhat-truffle5";
```

Comment:
This is needed to ensure that the type system pulls web3 and truffle artifacts into the hardhat import within the tests.